src/plotSearch/components/plotSearchSections/basicInfo/PlotSearchSiteEditCustomDetailedPlan.js

```diff
@@ -0,0 +1,280 @@
+              {'Osoite'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'address') || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaava'}
+            </FormTextTitle>
+            {get(currentCustomDetailedPlan, 'detailed_plan')
+              ? <ExternalLink
+                href={createPaikkatietovipunenUrl(`helreport/planpdfloader/?id=${get(currentCustomDetailedPlan, 'detailed_plan')}`)}
+                text={get(currentCustomDetailedPlan, 'detailed_plan')}
+              />
+              : <FormText>-</FormText>
+            }
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaavan viimeisin käsittelypvm'}
+            </FormTextTitle>
+            <FormText>
+              {formatDate(get(currentCustomDetailedPlan, 'detailed_plan_latest_processing_date')) || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaavan viimeisin käsittelypvm. selite'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'detailed_plan_latest_processing_date_note') || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kaavayksikön olotila'}
+            </FormTextTitle>
+            <FormText>{currentCustomDetailedPlan && getLabelOfOption(customDetailedPlanStateOptions, currentCustomDetailedPlan.state.id) || '-'}</FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kaavayksikön laji'}
+            </FormTextTitle>
+            <FormText>{currentCustomDetailedPlan && getLabelOfOption(customDetailedPlanTypeOptions, currentCustomDetailedPlan.type.id) || '-'}</FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kokonaisrakennusoikeus'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'rent_build_permission') || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Arvioitu rakentamisen valmius'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'preconstruction_estimated_construction_readiness_moment') || '-'}
+            </FormText>
+          </Column>
+          {infoLinks.length > 0 && (<Column small={12} medium={12} large={12} className="plot_search_target__info-links">
+            <Row>
+              <Column small={12} medium={4} large={4}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.INFO_LINK_DESCRIPTION}
```

Comment:
🔄 This component also suffers from having only some of the titles in the enum and some in the component source.